smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/ShapeValueGenerator.kt

```diff
@@ -292,17 +292,9 @@ class ShapeValueGenerator(
                 }
 
                 ShapeType.BYTE, ShapeType.SHORT, ShapeType.INTEGER,
-                ShapeType.LONG, ShapeType.DOUBLE, ShapeType.FLOAT -> writer.writeInline("\$L", node.value)
+                ShapeType.LONG, ShapeType.DOUBLE, ShapeType.FLOAT,
+                ShapeType.BIG_INTEGER, ShapeType.BIG_DECIMAL -> writer.writeInline("\$L", node.value)
 
-                ShapeType.BIG_INTEGER -> {
-                    writer.addImport(SwiftDependency.BIG.target)
-                    writer.writeInline("Array(String(\$L).utf8)", node.value)
-                }
-
-                ShapeType.BIG_DECIMAL -> {
-                    writer.addImport(SwiftDependency.BIG.target)
-                    writer.writeInline("Complex(\$L)", (node.value as Double).toBigDecimal())
-                }
```

Comment:
BigInteger and BigDecimal are unused codegen and are removed.  Those types will use Swift Int and Double, respectively.